semcore/tooltip/src/style/tooltip.shadow.css

```diff
@@ -72,6 +72,14 @@ SArrow[theme='warning'] {
   }
 }
 
+SArrow[bgColor][shadowColor] {
```

Comment:
I'm not sure it will be good to set them only when both are provided while it's two optional properties. Maybe we should provide it in one prop as object?

Reply:
`shadowColor` has a fallback to `bgColor`, so, here it will always be defined if `bgColor` is defined